src/DynamoPackagesUI/ViewModels/PackageManagerViewModel.cs

```diff
@@ -16,36 +16,29 @@ namespace Dynamo.DynamoPackagesUI.ViewModels
     public class PackageManagerViewModel
     {
         public const string PACKAGE_MANAGER_URL = "http://dynamopackagemanager.com.s3-website-us-east-1.amazonaws.com";
+        
         public string Address { get; set; }
 
-        internal PackageManagerCefHelper CefHelper { get; set; }
+        internal PackageManagerCommands CefHelper { get; set; }
```

Comment:
Property name should reflect the return type of the property, hence it should be named as Commands or something similar. What is the difference in two instances of Commands?